twitter-langchain/examples/chatbot/.env-local

```diff
@@ -0,0 +1,6 @@
+	OPENAI_API_KEY=
```

Comment:
nit: remove the left trailing whitespace in these lines

Reply:
roger that